lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -345,63 +390,54 @@ void OrphanedNodesManager::handleOp(const Ufe::SceneCompositeNotification::Op& o
             return;
         }
 
-        auto parentHier = Ufe::Hierarchy::hierarchy(parentItem);
-        if (!parentHier->hasChildren()) {
```

Comment:
This line was wrong because it ignored deactivated prims. Prims get deactivated when edited-as-Maya, so we would not process notifications correctly in all cases.